yogstation/code/game/gamemodes/battle_royale/battleroyale.dm

```diff
@@ -373,11 +379,11 @@ GLOBAL_VAR(final_zone)
 	name = "very cool tie (do not remove)"
 	desc = "Totally not just here for keeping track of kills."
 	var/datum/antagonist/battleroyale/last_hit
-	clothing_traits = (TRAIT_NODROP)
 	resistance_flags = INDESTRUCTIBLE //no escaping
 
 /obj/item/clothing/neck/tie/gamer/equipped(mob/user, slot)
 	. = ..()
+	ADD_TRAIT(src, TRAIT_NODROP, "I will kill you if you take this off somehow") //can't be a clothing trait because those get applied to the mob, not the item
```

Comment:
Shouldn't this be in initialize?

Reply:
... probably? i can't remember why i made it equipped in the first place